slices/apache2-utils.yaml

```diff
@@ -0,0 +1,30 @@
+package: apache2-utils
+
+essential:
+  - apache2-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libapr1t64_libs
+      - libaprutil1t64_libs
+      - libc6_libs
+      - libcrypt1_libs
+      - libssl3t64_libs
+    contents:
+      /usr/bin/ab:
+      /usr/bin/checkgid:
+      /usr/bin/fcgistarter:
+      /usr/bin/htcacheclean:
+      /usr/bin/htdbm:
+      /usr/bin/htdigest:
+      /usr/bin/htpasswd:
+      /usr/bin/logresolve:
+      /usr/bin/rotatelogs:
+      /usr/sbin/check_forensic:
+      /usr/sbin/httxt2dbm:
+      /usr/sbin/split-logfile:
```

Comment:
I think we should put `check_forensic` & `split-logfile` into a separate slice named `scripts`.
And the `scripts` slice should have the following essentials:
- base-file_bin
- coreutils_bins
- grep_bins
- grep_deprecated
- perl-base_bins

This is because `split-logfile` is a perl script and `check_forensic` is a shell script that uses mktemp, rm, cut, grep, join, xargs & egrep.